src/pages/Browse/index.tsx

```diff
@@ -133,8 +134,16 @@ export class BrowseBase extends React.Component<Props> {
     if (!file || !version) {
       return <Loading message={gettext('Loading content...')} />;
     }
-    if (file.type === 'image') {
-      return <ImageView content={file.content} mimeType={file.mimeType} />;
+    if (file.type === 'image' && file.downloadURL) {
+      return (
+        <div className={styles.Image}>
+          <img
+            alt=""
```

Comment:
I believe this is needed because of ESLint. Could you set it to `null` so that it does not appear in the generated HTML?

Reply:
Yes, that's why it's needed. Sure, I can set it to `null`.

Reply:
Nope, trying to set it to `null` still yields the ESLint error. It's either `""` or add a comment to disable the error.